kobuki_keyop/src/keyop_node.cpp

```diff
@@ -27,42 +27,23 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-/**
- * @file /kobuki_keyop/src/main.cpp
- *
- * @brief Executable code for the keyop node.
- *
- **/
+#include <cstdio>
+#include <memory>
 
-/*****************************************************************************
-** Includes
-*****************************************************************************/
+#include <rclcpp/rclcpp.hpp>
 
-#include "../include/keyop_core/keyop_core.hpp"
+#include "kobuki_keyop/keyop.hpp"
 
-/*****************************************************************************
-** Using
-*****************************************************************************/
+int main(int argc, char** argv)
+{
+  // Force flush of the stdout buffer.
```

Comment:
Just reading up on this method - might be more apt to say this configures the stream for 'no buffering'. On reading it as is without knowing the function beforehand I guessed incorrectly this was doing a one-time flush.

Reply:
Yeah, good idea.  See if the comment in 1e2e665 is more clear.